sdk/include/crt/mingw32/intrin_x86.h

```diff
@@ -89,6 +89,7 @@ __INTRIN_INLINE void* __cdecl memcpy(void* dest, const void* source, size_t num)
 
 /*** Memory barriers ***/
 
+#ifndef __clang__
```

Comment:
We dont have any intrinsics for clang?

Reply:
Rather, clang-cl has these builtin.

Reply:
ok...
I am not sure if this is good or bad, but in that case, these changes lgtm